### PR TITLE
resaved maps with tiled v1.4.3

### PR DESCRIPTION
**Pull request by HoraK-FDF**

### CODE REVIEW

File: graphics/sprites/monsters/terranite-mask.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
       <frame index="1" offsetX="-1" offsetY="-40"/>
     </animation>
     <animation direction="right">
-      <frame index="3" offsetX="-1" offsetY="-40"/>
+      <frame index="3" offsetX="1" offsetY="-40"/>
```
Comment (pazkero) [inline graphics/sprites/monsters/terranite-mask.xml:15]: This... doesn't belongs to this PR O.o

File: maps/002-3.tmx [hunk 2]
```diff
@@ -519,17 +518,17 @@
...
 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1565,1566,1567,1546,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
+0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1309,1310,1311,1290,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
 0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,541,542,543,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
-0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,1565,1566,1567,1565,1566,1567,1571,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,
```
Comment (pazkero) [inline maps/002-3.tmx:528]: There were some mapping changes.... *confused*

File: graphics/sprites/monsters/terranite-mask.xml (hunk 1 above)
Comment (HoraK-FDF) [reply graphics/sprites/monsters/terranite-mask.xml:15]: yeah that is the problem this gets included in every pull request i make.

File: maps/002-3.tmx (hunk 2 above)
Comment (HoraK-FDF) [reply maps/002-3.tmx:528]: never changed that map could the new tiled version have a bug?

**Review by HoraK-FDF — COMMENTED** [review]
sorry forgot that i removed this but its not a good way to remove the tile that must be fixed different.

**Pull request closed without merging**